signer/tests/integration/complete_deposit.rs

```diff
@@ -490,16 +490,22 @@ async fn complete_deposit_validation_fee_too_low() {
     // Different: We need to update the amount to be something that
     // validation would reject. To do that we update our database.
     //
-    // The fee rate in this test is fixed at 10.0 sats per vbyte and the tx
-    // size is 235 bytes, so we lose 2350 sats to fees. The amount here is
-    // chosen so that 2350 + 546 is greater than it.
-    let deposit_amount = 2895;
+    // The fee rate in this test is fixed at 10.0 sats per vbyte.
+    // The tx size is ~217 bytes, resulting in a fee of ~2170 sats.
+    //
+    // With the Bitcoin dust limit of 546 sats, we need: deposit_amount - fee < dust_limit
+    // So: deposit_amount - 2170 < 546
+    // Therefore: deposit_amount < 2716 will fail the dust check
+    //
+    // Using 2715 gives us 2715 - 2170 = 545 sats, which is just below the dust limit.
+    let deposit_amount = 2715;
```

Comment:
Since this may change, what about `tx_size * 10 + 546 - 1`?

Reply:
Great idea 👍 Changed it to be dynamic in [0cbf439](https://github.com/stacks-network/sbtc/pull/1512/commits/0cbf4396bbe65b8201a8e7d8da92847577acd3d0)

Reply:
Do we need to go from `info.tx_info.fee` to fee rate and back to tx fee (eg, some rounding thingy)? vs returning directly `info.tx_info.fee + DEPOSIT_DUST_LIMIT - 1`